scripts/color-system-docs.js

```diff
@@ -1,8 +1,10 @@
 const {resolve: resolvePath} = require('path');
 const {writeFileSync} = require('fs-extra');
-const roleVariants = require('@shopify/polaris-tokens/formats/utils/color-factory/configs/base');
+const {
+  config: roleVariants,
```

Comment:
Ah, this is where I was getting hung up

Reply:
yeah this is weird, was not expecting this to change with the TS build